### PR TITLE
Update workflow for aligned groups creation - add, remove, merge

### PR DESCRIPTION
**Pull request by irina060981**
For the ussue https://github.com/alpheios-project/alignment-editor-new/issues/24

Realized workflow for
* add items to new and before created groups
* remove items from new and before created groups
* merge groups
* update css classes:
      -  hover for non-grouped `:hover`
      - already in a saved group (without hover) `.alpheios-token-grouped`
      - already in saved group (hover) `.alpheios-token-selected`
      - in active group `.alpheios-token-clicked`
      - in active group selected first `.alpheios-token-clicked-first`

Also I changed some methods names for better consistency

### CODE REVIEW

File: src/lib/data/alignment.js [hunk 1]
```diff
@@ -63,57 +63,110 @@ export default class Alignment {
...
+      console.error(this.l10n.getMsg('ALIGNMENT_ERROR_ADD_TO_ALIGNMENT'))
+    }
+  }
+
+  removeFromAlignmentGroup (token) {
+    if (this.activeAlignmentGroup[token.textType]) {
+      this.activeAlignmentGroup.remove(token)
+      this.removeFromAlignmentIds(token.idWord)
     } else {
       console.error(this.l10n.getMsg('ALIGNMENT_ERROR_ADD_TO_ALIGNMENT'))
```
Comment (balmas) [inline src/lib/data/alignment.js:94]: very minor point, but the error on line 94 should probably be specific to removing not adding a token

File: src/lib/data/alignment-group.js [hunk 2]
```diff
@@ -34,10 +47,27 @@ export default class AlignmentGroup {
...
+    this.origin.push(...tokensGroup.origin)
+    this.target.push(...tokensGroup.target)
+
+    tokensGroup.origin.forEach(idWord => {
+      this.steps.push({ textType: 'origin', id: idWord, type: 'merge' })
+    })
+
+    tokensGroup.target.forEach(idWord => {
+      this.steps.push({ textType: 'target', id: idWord, type: 'merge' })
+    })
```
Comment (balmas) [inline src/lib/data/alignment-group.js:71]: I agree that the way you manage the steps will have to be updated to deal with undo of a merge -- i.e you'll need a way to indicate which actions should be undone/redone together.

File: src/lib/data/alignment.js (hunk 1 above)
Comment (irina060981) [reply src/lib/data/alignment.js:94]: oh, thank you for the catch

File: src/lib/data/alignment-group.js (hunk 2 above)
Comment (irina060981) [reply src/lib/data/alignment-group.js:71]: yes, I need to try some variants

**Comment by irina060981** [comment]
>another thought -- I think there is another business logic in there now that it's probably time to write some tests?

yes, I agree. It is time to add frunctions description and tests

**Pull request merged**